changelog.md

```diff
@@ -10,6 +10,7 @@
 
 ### Features
 
+* [1518](https://github.com/zeta-chain/node/pull/1518) - Avoid duplicate keysign if an outTx is already pending
```

Comment:
Can we move it to fix?

Reply:
[commit](https://github.com/zeta-chain/node/pull/1518/commits/0b42c7b456b475c9cf0e40aa62a2875d5da966ff)